bubblewrap.c

```diff
@@ -1624,9 +1639,30 @@ main (int    argc,
       exit (0); /* Should not be reached, but better safe... */
     }
 
+  /* Child, in sandbox, privileged in the parent *or* in the user namespace.
+   *
+   * NOTE: This is always ptrace:able in the case of user namespaces
+   * (due to all parent namespaces having CAP_SYS_PTRACE in the child
+   * namespaces), but it should be ok as it has no permissions in the
+   * parent namespace.
+   */
+
   if (opt_info_fd != -1)
     close (opt_info_fd);
 
+  if (is_privileged && opt_unshare_user)
+    {
+      /* We have to be dumpable for the parent to be able to set the
+         uid map for us. This enables ptracing for the child, but that
+         is not really a major*/
```

Comment:
major...problem?  I'd say something like "is believed safe, as at this point we should have no extra privileges".  Right?